packages/prisma/seed.ts

```diff
@@ -606,6 +606,81 @@ async function main() {
       },
     ]
   );
+
+  await prisma.team.create({
```

Comment:
Creating an unpublished team with event types, as well as an org with event types and users with event types, to run e2e and have available to test app workflows when publishing them

Reply:
We need to skip if database entry already exists, I am getting this error: 
<img width="982" alt="Screenshot 2023-07-28 at 16 04 34" src="https://github.com/calcom/cal.com/assets/30310907/c47e15e8-43ac-4694-8a26-6936e90ab662">

And Foobar Team is now there several times, no slug yet so it doesn't throw any unique constraint
<img width="765" alt="Screenshot 2023-07-28 at 16 07 14" src="https://github.com/calcom/cal.com/assets/30310907/d5f5c74e-6375-4354-bd0e-cb5b561fe740">

Reply:
Oh so are you running the seed without resetting the DB? 🤔

Reply:
I ran yarn dx, I think this always runs seed

Reply:
Corrected, now using seed utils with some adjustments